src/db/mod.rs

```diff
@@ -20,10 +20,10 @@ pub use self::idgen::id_generator;
 pub static LOCAL_ACCOUNT_DOMAIN: &'static str = "";
 
 /// Convenient type alias for the postgres database pool so we don't have to type this out.
-type Pool = r2d2::Pool<ConnectionManager<PgConnection>>;
+pub type Pool = r2d2::Pool<ConnectionManager<PgConnection>>;
 
 /// Type alias for the pooled connection.
-type PooledConnection = r2d2::PooledConnection<ConnectionManager<PgConnection>>;
+pub type PooledConnection = r2d2::PooledConnection<ConnectionManager<PgConnection>>;
```

Comment:
It doesn't seem like `PooledConnection` needs to be exposed for these changes to build.  Did you have a future change in mind that would require it to be public?

Reply:
I just made it public while the other one was, I guess? haha

Reply:
I don't have a preference either way, just curious :)